engine/npc.py

```diff
@@ -0,0 +1,17 @@
+from engine.point import Point
+from random import choice
+
+possible_names = ['steve', 'mark', 'jill', 'edna']
```

Comment:
It is a standard convention that when you make a variable outside of a class like this, to make it ALL CAPS
POSSIBLE_NAMES = []